examples/gke-security-best-practices/builder/platform-uploads/tmp/acme-recovery-user.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa {SSH Key Base64 goes here}
```

Comment:
From where?

Reply:
this is not required, I will remove this step.